src/styles/stars.css

```
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  height: 16px;
+  width: 16px;
+  margin-left: 4px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
+
+.stars--4 .stars__star:nth-child(1) {
```

don't increase specificity if it's unnecessary
```suggestion
.stars__star:nth-child(1) {
```